l10n_it_vat_registries/__manifest__.py

```diff
@@ -7,8 +7,8 @@
 # License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
 
 {
-    'name': 'Italian Localization - VAT Registries',
-    'version': '10.0.1.2.3',
+    'name': 'ITA - VAT Registries',
```

Comment:
Dato che non è traducibile, andrebbe in italiano.

Reply:
Ah mi manca sempre qualche pezzetto, ora ho corretto.
Grazie della review!

Reply:
Figurati! :wink: :+1: